### ISSUE TITLE
Project still maintained?

### ISSUE THREAD

**Issue by forki** [pd29/vscode-fantomas-fmt#25]
Hi,

I noticed that this project did not get any updates anymore. I assume you are using the fantomas that ships with Ionide!?
If so, could you please disable the addin from the gallery? It's the first on shown if you search for "Fantomas".
If not, then please excuse my question. Just trying to find out what's the easiest thing for users to get started.

Steffen

**Comment by joshuapassos** [comment]
I think that this project is a good choice because you can to set specific settings without create `.editorconfig`, but is necessary update the repository

**Comment by nojaf** [comment]
I don't believe that argument is still valid as stylistic options can no longer be passed via CLI arguments.
For quite some time now, `.editorconfig` is the only way to specify this.
